Clients/src/presentation/pages/DashboardOverview/README.md

```
@@ -0,0 +1,248 @@
+```tsx
+import { DashboardWithToggle } from './DashboardOverview/DashboardWithToggle';
+import OriginalDashboard from './DashboardOverview';
+
+function App() {
+  return <DashboardWithToggle OriginalDashboard={OriginalDashboard} />;
+}
+```
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)